pom.xml

```diff
@@ -46,6 +46,11 @@
             <artifactId>spring-boot-starter-test</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.modelmapper</groupId>
+            <artifactId>modelmapper</artifactId>
```

Comment:
I have checked this library dependencies and it seems to have none that could be a problem (actually they have barely any). So it seems like a nice addition.